db.go

```diff
@@ -213,6 +215,76 @@ func (db *DB) Get(ro *ReadOptions, key []byte) ([]byte, error) {
 	return C.GoBytes(unsafe.Pointer(value), C.int(vallen)), nil
 }
 
+// GetMany returns the values associated with keys from the database.
+//
+// GetMany fills in the entries of values consistent with Get():
+// - If keys[i] exists in the database with non-zero value, values[i] holds its non-zero len []byte slice.
+// - If keys[i] does not exist in the database, values[i] == nil.
+// - If keys[i] does exist, but its value is zero-length, values[i] == []byte{}.
+// - If there's an error looking up keys[i], values[i] == nil.
+//
+// The keys byte slice may be reused safely. Go takes a copy of
+// them before returning.
+func (db *DB) GetMany(ro *ReadOptions, keys [][]byte) ([][]byte, []error) {
+	if db.closed {
+		panic(ErrDBClosed)
+	}
+	if len(keys) == 0 {
+		return nil, nil
+	}
+
+	var packedKeysBuffer bytes.Buffer
+	cKeyLens := make([]C.size_t, len(keys))
+	for i := range keys {
+		packedKeysBuffer.Write(keys[i])
+		cKeyLens[i] = C.size_t(len(keys[i]))
+	}
+	packedKeysBytes := packedKeysBuffer.Bytes()
```

Comment:
return error if `len(packedKeysBytes)` (if all keys are 0-bytes) ?
otherwise you'll panic l.249